app/uk/gov/hmrc/nisp/controllers/auth/AuthAction.scala

```diff
@@ -133,8 +136,8 @@ class VerifyAuthActionImpl @Inject()(override val authConnector: NispAuthConnect
       } recover {
       case _: NoActiveSession |
            _: InsufficientConfidenceLevel |
-           _: UnsupportedAuthProvider => Redirect(ApplicationConfig.verifySignIn, parameters).withSession(
-        SessionKeys.redirect -> ApplicationConfig.postSignInRedirectUrl,
+           _: UnsupportedAuthProvider => Redirect(applicationConfig.verifySignIn, parameters).withSession(
+        SessionKeys.redirect -> applicationConfig.postSignInRedirectUrl,
         SessionKeys.loginOrigin -> "YSP"
```

Comment:
I have got to assume that it is not in app conf as it is a value that will not change. It could be something to investigate as part of the refactor phase.